preprocessing/nextclade/src/loculus_preprocessing/processing_functions.py

```diff
@@ -64,8 +84,11 @@ def format_authors(authors: str) -> bool:
         first_name = " ".join(
             [f"{name}." if len(name) == 1 else name for name in first_name.split(" ")]
         )
-        loculus_authors.append(f"{last_name}, {first_name}")
-    return "; ".join(loculus_authors) + ";"
+        # Capitalize last name and first name
+        loculus_authors.append(
+            f"{convert_to_title_case(last_name)}, {convert_to_title_case(first_name)}"
```

Comment:
I don't think we should fiddle with case except in the very specific situation that the entire authors field is in uppercase, suggesting a formatting issue. Here are some cases which would be messed up by the current logic:

`Ronald McDonald`
`The RECOVERY Consortium`